portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java

```diff
@@ -197,18 +197,18 @@ public void testHasValue() {
 
 	@Test
 	public void testIsJSONArray() {
-		Assert.assertFalse(
-			"null is not a JSON array", JSONUtil.isJSONArray(null));
-
-		Assert.assertFalse(
-			"\"\" is not a JSON array", JSONUtil.isJSONArray(""));
+		Assert.assertTrue(
+			"null is an empty JSON array", JSONUtil.isJSONArray(null));
 
-		Assert.assertFalse(
-			"{} is not a JSON array", JSONUtil.isJSONArray("{}"));
+		Assert.assertTrue(
+			"\"\" is an empty JSON array", JSONUtil.isJSONArray(""));
```

Comment:
@alecsloan we should use `StringPool.BLANK` here.
```suggestion
			"\"\" is an empty JSON array", JSONUtil.isJSONArray(StingPool.BLANK));
```